src/client/pps/pps.proto

```diff
@@ -68,11 +68,19 @@ message CronInput {
   google.protobuf.Timestamp start = 5;
 }
 
+message GithubInput {
+  string name = 1;
+  string url = 2 [(gogoproto.customname) = "URL"];
+  string branch = 3;
+  string commit = 4;
+}
```

Comment:
This should be called a Git input, even if it only works for Github right now eventually we'll want it to work for all git repos. Also I was under the impression that it would already work for other git hosts. Is that true?

Reply:
[The library I used to handle the webhooks supports bitbucket and gitlab](https://github.com/go-playground/webhooks/tree/v3.4.0) but for us to support these we'd need to add those handlers to the githook server. Since there was no ask for those, I didn't add them. 

That said, I can change the name.

Reply:
Yeah, there's no need to implement those now, but particularly for protos we should always try to minimize the chances that we'll have to rename something since that causes all sorts of migration headaches.